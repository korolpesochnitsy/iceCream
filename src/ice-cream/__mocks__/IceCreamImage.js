import React from 'react';

const IceCreamImage = ({ iceCreamId }) => (
  <img src={`ice-cream-${iceCreamId}.svg`} alt="" />
);

export default IceCreamImage;
