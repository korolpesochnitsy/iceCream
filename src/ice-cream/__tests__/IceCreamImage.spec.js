import React from 'react';
import { render, cleanup } from '@testing-library/react';
import IceCreamImage from '../IceCreamImage';

describe('IceCreamImage', () => {
  afterEach(cleanup);
  it('should load and display the ice cream image', () => {
    const { container } = render(<IceCreamImage iceCreamId={12} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
