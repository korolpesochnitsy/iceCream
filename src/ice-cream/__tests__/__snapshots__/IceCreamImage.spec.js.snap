// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IceCreamImage should load and display the ice cream image 1`] = `
<img
  alt=""
  src="/ice-cream-images/ice-cream-12.svg"
/>
`;
